vendor/assets/javascripts/jquery.infinitescroll.js

```diff
@@ -0,0 +1,827 @@
+/*global jQuery: true */
```

Comment:
Normally, if we want to use 3rd party js or css lib without any modification or customization. Putting it in vendor folder is good choice.